src/openloco/windows/industry_list.cpp

```diff
@@ -1,16 +1,1323 @@
+                widthOffset = 138;
+
+                gfx::draw_string_494C78(*dpi, xPos, yPos, colour::black, string_ids::build_cost, &args);
+            }
+
+            auto xPos = self->x + 3;
+            auto yPos = self->y + self->height - 13;
+            auto width = self->width - 19 - widthOffset;
+
+            gfx::draw_string_494BBF(*dpi, xPos, yPos, width, colour::black, string_ids::white_stringid2, &industryObj->name);
+        }
+
+        // 0x0045843A
+        static void on_mouse_up(window* self, widget_index widgetIndex)
+        {
+            switch (widgetIndex)
+            {
+                case common::widx::close_button:
+                    WindowManager::close(self);
+                    break;
+
+                case common::widx::tab_industry_list:
+                case common::widx::tab_new_industry:
+                    common::switchTab(self, widgetIndex);
+                    break;
+            }
+        }
+
+        //0x00458966
+        static void on_scroll_mouse_down(ui::window* self, int16_t x, int16_t y, uint8_t scrollIndex)
+        {
+            int16_t xPos = (x / 122);
+            int16_t yPos = (y / 112) * 5;
+            auto index = xPos + yPos;
+
+            for (auto i = 0; i < self->var_83C; i++)
+            {
+                auto rowInfo = self->row_info[i];
+                index--;
+                if (index < 0)
+                {
+                    self->row_hover = rowInfo;
+                    byte_525FC7 = static_cast<uint8_t>(rowInfo);
+
+                    int32_t pan = (self->width >> 1) + self->x;
+                    loc16 loc = { xPos, yPos, static_cast<int16_t>(pan) };
+                    audio::play_sound(audio::sound_id::click_down, loc, pan);
+                    break;
+                }
+            }
+        }
+
+        // 0x00458721
+        static void on_scroll_mouse_over(ui::window* self, int16_t x, int16_t y, uint8_t scrollIndex)
+        {
+            auto xPos = (x / 122);
+            auto yPos = (y / 112) * 5;
+            auto index = xPos + yPos;
+            uint16_t rowInfo = 0xFFFF;
+
+            for (auto i = 0; i < self->var_83C; i++)
+            {
+                rowInfo = self->row_info[i];
+                index--;
+                if (index < 0)
+                    break;
+            }
+            self->var_846 = rowInfo;
+            self->invalidate();
+            auto string = string_ids::buffer_337;
+
+            if (rowInfo == 0xFFFF)
+                string = string_ids::null;
+
+            if (string != string_ids::empty)
+            {
+                if (string == self->widgets[common::widx::frame].tooltip)
+                {
+                    if (rowInfo == self->var_85C)
+                        return;
+                }
+            }
+            self->widgets[common::widx::frame].tooltip = string;
+            self->var_85C = rowInfo;
+            call(0x004C87B5);
```

Comment:
`tooltip::closeAndReset();` https://github.com/OpenLoco/OpenLoco/pull/462/ depends on which PR is merged first

Reply:
#462 has been merged, so `tooltip::closeAndReset();` can be used after a rebase.